sympy/physics/units/tests/test_unit_system_cgs_gauss.py

```diff
@@ -31,6 +31,8 @@ def test_conversion_to_from_si():
     assert convert_to(newton, dyne, SI) == 10**5*dyne
     assert convert_to(newton, dyne, cgs_gauss) == 10**5*dyne
 
+def test_angstrom_to_cgs_gauss():
+    assert convert_to(angstrom, centimeter, cgs_gauss) == 1*centimeter/10**8
```

Comment:
maybe this can be added to the previous test function (do we need a test function just for angstrom)?